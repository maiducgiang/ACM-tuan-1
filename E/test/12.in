100
140522259 918039521
541517398 200915594
413378326 982577345
235601549 411166146
908667608 722797491
183933006 282915706
962534459 974052747
551352639 560221891
332080524 504704888
349553880 792460842
666756702 283075237
767697896 579873597
18479860 369251201
817324538 529581128
46735660 24507011
212498645 467819233
672698964 993268790
946728813 241171369
486170680 62866722
96098345 484055420
541660759 845424362
99737539 28758393
396096757 465503664
167855704 548950254
44083625 224896366
793596467 126945740
157961126 93053742
538524818 26910718
380752321 502572849
982761595 190935743
449647255 220237531
781757891 338854092
409058564 577024856
335473664 89930385
234960156 807163104
31435347 354075988
559994090 960446729
863488648 73322100
140732928 190092863
636598895 4632687
440729505 953754919
825000353 911359573
572857047 250019556
965432379 407848232
336055255 858006730
434734661 618808888
985446108 481273935
58427328 52994573
739932968 601923312
499196995 459732448
195757809 382435917
948323241 157234517
992744057 598683664
226180579 940686567
341317203 337744371
888602085 221788710
743371240 964849238
908344094 594258889
282178016 141956356
679929990 181233001
840788214 971220556
489179169 351526305
836799750 200273830
530303407 306274109
306493071 140559535
466711422 206509157
276622440 445445554
532768283 664376470
470954924 793504324
508269685 647970091
868045580 211136969
314725182 432720670
658807834 44474786
681378348 916283001
757627464 497707665
658603290 282330845
97006211 926881354
646829933 866795534
843392033 568877475
803599327 275546446
702705092 885069200
114084010 557903584
732546199 693508502
329998715 326279460
412017645 258149627
635524955 24062711
250164871 215189336
550466050 185301454
747828046 658203453
177376578 85367006
234035861 822689782
641643725 586090771
281346792 949481062
407238409 718442975
617481748 842662568
883315146 725663621
22314376 374471746
743382297 262781077
484553043 909350421
176472013 51052384
