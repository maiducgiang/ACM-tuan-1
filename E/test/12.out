7
2
3
2
2
2
2
2
2
3
2
2
20
2
2
3
2
2
2
6
2
2
2
4
6
2
2
2
2
2
2
2
2
2
4
12
2
2
2
2
3
2
2
2
3
2
2
2
2
2
2
2
2
5
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
10
2
2
2
2
5
2
2
2
2
2
2
2
2
4
2
4
2
2
2
17
2
2
2
